app/app.js

```diff
@@ -155,18 +155,18 @@ module.exports = async (options) => {
   })
 
   // Component example preview
-  app.get('/components/:componentName/:exampleName*?/preview', function (req, res, next) {
+  app.get('/components/:componentName/:exampleName?/preview', function (req, res, next) {
```

Comment:
TypeScript flagged this bit `:exampleName*?` not matching `req.params.exampleName`

* ❌ `/components/:componentName/:exampleName*?`

But we could have changed it to either:

1. ✅ `/components/:componentName/*?`
2. ✅ `/components/:componentName/:exampleName?`

Which would have matched:

**/components/accordion
/components/accordion/example1
/components/accordion/example2
/components/accordion/example3**

So kept the one with `:exampleName` 👍

Reply:
**Note:** It worked fine before, but didn't match the documented type declarations